hub/config/src/main/java/uk/gov/ida/hub/config/domain/TransactionConfigEntityData.java

```diff
@@ -110,6 +110,10 @@
     @JsonProperty
     protected boolean usingMatching = true;
 
+    @Valid
+    @JsonProperty
+    protected boolean eidasProxyNode;
```

Comment:
I think we should have a default value of `false` for this. Otherwise all current transactions will have invalid config the moment this is merged since this is a non-nullable field. Just like `usingMatching ` above has a default of `true` to avoid this situation.

Reply:
small b booleans are false by default, which is the required default state in this case. There is a test that checks this condition i.e. eidasProxyNode is never set to a value.

Reply:
Oh you're right! It will always be `false` by default. OK that looks then :)